src/handlers/mentions.rs

```diff
@@ -109,6 +132,11 @@ pub(super) async fn handle_input(
             None => write!(result, "Some changes occurred in {to_mention}").unwrap(),
         }
         if !cc.is_empty() {
+            let cc: Vec<String> = if input.has_bors_commit {
+                cc.iter().map(|s| format!("`{s}`")).collect()
```

Comment:
On Zulip there was a proposal to also mark the PR as draft, to prevent it from being merge.

Ideally we'd want a scheme where the pings definitely happen when the PR gets approved or lands, but I am not sure if that is possible.